tests/regression/37-congruence/01-simple.c

```diff
@@ -0,0 +1,28 @@
+//PARAM: --enable ana.int.congruence --enable ana.int.interval --disable ana.int.def_exc --disable ana.int.enums
+
+int main() {
+	int a = 1;
+	int b = 2;
+	int c = 3;
+	int d = 4;
+	int e = 0;
+
+	while (d < 9) {
+		b = 2 * a;
+		d = d + 4;
+		e = e - 4 * a;
+
+		a = b - a;
+		c = e + d;
+	}
+
+	a = d / 2;
+	b = d % 2;
+
+	assert (c == 4); // UNKNOWN!
+	assert (d == 12); // UNKNOWN
```

Comment:
Missing "!"

Reply:
This question came up a few weeks ago.
To our knowledge the difference is that 'UNKNOWN!' expects the variable to be definitely unknown (which means it should not have a precise value like 'c = 4' or in the congruence domain: c = {4} ((4,0) in our implementation). In the example above we figured that there is no way that c ever becomes a concrete val after a fixed-point iteration.
You're right though, we should definitely check the other tests again as our implementations and test cases changed quite frequently.